docker-wordpress-mysql/azuredeploy-parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "newStorageAccountName": {
+            "value": "unqiueStorageAccount"
+        },
+	"location": {
+	    "value": "West US"
+	},
+        "mysqlPassword": {
+            "value": "password"
+        },
+        "adminUsername": {
+            "value": "userName"
+        },
+        "adminPassword": {
+            "value": "password"
+        },
+        "dnsNameForPublicIP": {
+            "value": "uniqueDNS"
+        }
```

Comment:
I'd prefer to have as few parameters as possible. If somebody wants more options they can fork the template. Hardwire an UbuntuOSversion that you think is the most likely to work with the extension and leave it out of the parameters file. You can see how I wrote the template here:  https://github.com/johngossman/azure-quickstart-templates/blob/master/docker-wordpress/azuredeploy.json